routes/web.php

```diff
@@ -18,5 +18,5 @@
     return view('welcome');
 });
 
-Route::get('/invoices', [InvoiceController::class, 'index']);
-Route::get('/invoices/{id}', [InvoiceController::class, 'show']);
+Route::get('/invoices', [InvoiceController::class, 'index'])->name('invoices.index');
+Route::get('/invoices/{id}', [InvoiceController::class, 'show'])->name('invoices.show');
```

Comment:
In class, I named these routes using `invoice.index` and `invoice.show` respectively. Notice that "invoice" is singular instead of "invoices". It doesn't matter what these names are. The important part is that the name you give the route matches what you pass to the `route()` function.